<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MessageTipForm</class>
 <widget class="QDialog" name="MessageTipForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>262</width>
    <height>81</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>272</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>新消息(0)</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#MessageTipForm{
background-color: rgb(255, 255, 255);
     border-width: 1px;
     border-style: solid;
     border-color:#D1D1D1;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QWidget" name="widget_header" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>36</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>260</width>
         <height>36</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget#widget_header{
	image: url(:/res/MessageTipForm/bg.png);
}
</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_msgTotal">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>新消息(0)</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>88</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>6</number>
       </property>
       <item>
        <widget class="QListWidget" name="listWidget_msg">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>2</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QListWidget#listWidget_msg{
     border-width: 0px;
     border-style: solid;
     border-color:#D1D1D1;
background-color: rgb(255, 255, 255);
}

QListWidget#listWidget_msg::item{
    border:none;
       height: 34px;
}

QListWidget#listWidget_msg::item:hover{
    background:rgb(221,242,254)
}</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="leftMargin">
          <number>7</number>
         </property>
         <property name="rightMargin">
          <number>7</number>
         </property>
         <item>
          <widget class="QPushButton" name="pushButton_ignoreAll">
           <property name="styleSheet">
            <string notr="true">QPushButton#pushButton_ignoreAll {
     border-width: 0px;
     border-style: solid;
	 color:#3079CA;
} 
QPushButton#pushButton_ignoreAll::hover{
	text-decoration:underline;
}
</string>
           </property>
           <property name="text">
            <string>忽略全部</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_openAll">
           <property name="styleSheet">
            <string notr="true">QPushButton#pushButton_openAll {
     border-width: 0px;
     border-style: solid;
     color:#3079CA;
} 
QPushButton#pushButton_openAll::hover{
	text-decoration:underline;
}</string>
           </property>
           <property name="text">
            <string>打开全部</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
